<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellingPropertiesDialog</class>
 <widget class="QDialog" name="SpellingPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spelling Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1_3">
     <property name="text">
      <string>Project &amp;Language:</string>
     </property>
     <property name="buddy">
      <cstring>spellingComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="spellingComboBox">
     <property name="toolTip">
      <string>Select the project's language</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project &amp;Word List:</string>
     </property>
     <property name="buddy">
      <cstring>pwlEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="pwlEdit">
     <property name="toolTip">
      <string>Enter the filename of the project word list</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project E&amp;xclude List:</string>
     </property>
     <property name="buddy">
      <cstring>pelEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="pelEdit">
     <property name="toolTip">
      <string>Enter the filename of the project exclude list</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="pwlButton">
     <property name="toolTip">
      <string>Select the project word list file via a file selection dialog</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="pelButton">
     <property name="toolTip">
      <string>Select the project exclude list file via a file selection dialog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spellingComboBox</tabstop>
  <tabstop>pwlEdit</tabstop>
  <tabstop>pwlButton</tabstop>
  <tabstop>pelEdit</tabstop>
  <tabstop>pelButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpellingPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpellingPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
